environments/cooker.json

```diff
@@ -16,5 +16,7 @@
     "business-unit": "Platforms",
     "infrastructure-support": "modernisation-platform@digital.justice.gov.uk",
     "owner": "Modernisation Platform: modernisation-platform@digital.justice.gov.uk"
-  }
+  },
+  "go-live-date": "",
```

Comment:
If this is blank, does it serve any purpose? Could we just set the same go-live date as other platform accounts here?

Reply:
Well, it's not live so doesn't have a go live date, same with example.